### PR TITLE
 Implemented query builder for CAL source and fixed adapater bug

### PR DESCRIPTION
**Pull request by capri-xiyue**
- fixed missing adapterType bug

- fixed typo

### CODE REVIEW

File: pkg/reconciler/events/cloudauditlog/filterbuilder.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+func (fb *FilterBuilder) SetResourceName(resourceName string) {
+	fb.resourceName = resourceName
+}
+
+func (fb *FilterBuilder) GetFilterQuery() string {
+	var filters []string
+	if fb.methodName != "" {
+		filters = append(filters, fmt.Sprintf("%s=%q", methodKey, fb.methodName))
```
Comment (ian-mi) [inline pkg/reconciler/events/cloudauditlog/filterbuilder.go]: Can this use a typed K/V struct which shares this formatting?

Comment (ian-mi) [inline pkg/reconciler/events/cloudauditlog/filterbuilder.go]: Filter values should probably be quoted.

**Review by ian-mi — CHANGES_REQUESTED** [review]
Thanks for the fixes!

File: pkg/reconciler/events/cloudauditlog/filterbuilder.go (hunk 1 above)
Comment (capri-xiyue) [reply pkg/reconciler/events/cloudauditlog/filterbuilder.go]: %q	a double-quoted string safely escaped with Go syntax 
FYI: https://golang.org/pkg/fmt/
The filter values already got double-quoted

**Pull request merged**